poonam/JS/Assignment14_js/que1.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Student marks</title>
+    </head>
+    <body>
+        <script language="javascript">
+            class student{
+            constructor(RollNo,Name){
+            this.RollNo=RollNo;
+            this.Name=Name;
+            }
+            display(){
+            document.write("<br>Inside Display");
+            document.write("<br>Roll No "+this.RollNo);
+            document.write("<br>Name "+this.Name);
+            }
+            }
+            //Marks is a derived class/child class
+            class Marks extends student{
+            //Construcor Overrriding
+            constructor(RollNo,Name,Marks){
+            super(RollNo,Name);
+            this.RollNo=RollNo;
+            this.Name=Name;
```

Comment:
Remove this Because it's already defined in Base Class so we don't need to Defined again.
```suggestion
            this.RollNo=RollNo;
            this.Name=Name;
```